# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/ftp/FTPChannelParent.cpp
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/protocol/ftp/FTPChannelParent.cpp
--- a/netwerk/protocol/ftp/FTPChannelParent.cpp	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/protocol/ftp/FTPChannelParent.cpp	Tue Feb 12 16:08:25 2019 +0000
@@ -164,11 +164,7 @@
   rv = ftpChan->ResumeAt(aStartPos, aEntityID);
   if (NS_FAILED(rv)) return SendFailedAsyncOpen(rv);
 
-  if (loadInfo && loadInfo->GetEnforceSecurity()) {
-    rv = ftpChan->AsyncOpen2(this);
-  } else {
-    rv = ftpChan->AsyncOpen(this, nullptr);
-  }
+  rv = ftpChan->AsyncOpen(this);
 
   if (NS_FAILED(rv)) return SendFailedAsyncOpen(rv);
 